src/content/en/2019/third-parties.md

```diff
@@ -0,0 +1,167 @@
+
+<insert graphic of metric 05_03>
+
+Several other amusing factoids jump out from this data. Tracking pixels (image requests to analytics domains) make up 1.6% of all network requests, six times as many video requests are to social networks like Facebook and Twitter than dedicated video providers like YouTube and Vimeo (presumably because the default YouTube embed consists of HTML and a preview thumbnail but not an autoplaying video), and there are still more requests for first-party images than all scripts combined.
+
+### Request Count
+
+49% of all requests are third-party. At 51%, first-party can still narrowly hold on to the crown in 2019 of comprising the majority of the web resources. Given that half of all the requests are third-party yet a small set of pages do not include any at all, the most active third-party users must be doing quite a bit more than their fair share. Indeed, at the 75th, 90th, and 99th percentiles we see nearly all of the page being comprised of third-party content. In fact, for some sites heavily relying on distributed WYSIWYG platforms like Wix and SquareSpace, the root document might be the sole first-party request!
+
+<insert graphic of metric 05_11>
+
+The number of requests issued by each third-party provider also varies considerably by category. While analytics are the most widespread third-party category across websites, they account for only 7% of all third-party network requests. Ads, on the other hand, are found on nearly 20% fewer sites yet make up 25% of all third-party network requests. Their outsized resource impact compared to their popularity will be a theme we continue to uncover in the remaining data.
+
+### Byte Weight
+
+While 49% of requests are third-party, their share of the web in terms of bytes is quite a bit lower at only 28%. The same goes for the breakdown by multiple resource types. Third-party fonts make up 72% of all fonts, but they’re only 53% of font bytes; 74% of HTML requests, but only 39% of HTML bytes; 68% of video requests, but only 31% of video bytes. All this seems to suggest third-party providers are responsible stewards who keep their response sizes low, and, for the most part, that is in fact the case until you look at scripts.
```

Comment:
Does bytes here refer to just the bytes downloaded on the first response or does it include bytes from subsequent document.write()s from the initial response. It's not uncommon for ads to have 10ish long request chain.

Reply:
bytes everywhere in this refers to the number of bytes transferred over the wire, the resource type classification for the bytes is whatever the resource was as a whole.

i.e. bytes for inline script of a document are classified as HTML and bytes for HTML in a `document.write` string of a script file are classified as script


maybe worth putting this up with definitions?

Reply:
I think it's ok as-is for the purpose of this chapter. If we need to clarify specific metrics we can do that on the Methodology page (#202) or the [forum](discuss.httparchive.org).